crates/holochain_persistence_lmdb/src/eav/lmdb.rs

```diff
@@ -0,0 +1,299 @@
+use holochain_persistence_api::{
+    cas::content::AddressableContent,
+    eav::{
+        Attribute, EavFilter, EaviQuery, EntityAttributeValueIndex, EntityAttributeValueStorage,
+    },
+    error::{PersistenceError, PersistenceResult},
+    reporting::{ReportStorage, StorageReport},
+};
+// use kv::{Config, Manager, Store, Error as KvError};
+use rkv::{
+    error::StoreError, DatabaseFlags, EnvironmentFlags, Manager, Rkv, SingleStore, StoreOptions,
+    Value,
+};
+use std::{
+    collections::BTreeSet,
+    fmt::{Debug, Error, Formatter},
+    marker::{PhantomData, Send, Sync},
+    path::Path,
+    sync::{Arc, RwLock},
+};
+use uuid::Uuid;
+
+const EAV_BUCKET: &str = "EAV";
+const MAX_SIZE_BYTES: usize = 104857600; // TODO: Discuss what this should be, currently 100MB
+
+#[derive(Clone)]
+pub struct EavLmdbStorage<A: Attribute> {
+    id: Uuid,
+    store: SingleStore,
+    manager: Arc<RwLock<Rkv>>,
+    attribute: PhantomData<A>,
+}
+
+impl<A: Attribute> EavLmdbStorage<A> {
+    pub fn new<P: AsRef<Path> + Clone>(db_path: P) -> EavLmdbStorage<A> {
+        let cas_db_path = db_path.as_ref().join("eav").with_extension("db");
```

Comment:
```suggestion
        let eav_db_path = db_path.as_ref().join("eav").with_extension("db");
```

This is an argument in favor of @AshantiMutinta's DRY suggestion below ;)